### PR TITLE
Remove clondeNode IE8 workaround

### LINKED ISSUE #65: Remove cloneNode code?
image

https://github.com/canjs/can-view-target/blob/9cb4a0a8647419429152764d43263bcbbd12990a/can-view-target.js#L85-L128

### PR DESCRIPTION
**Pull request by cherifGsoul**
- Remove `clondeNode` function needed for IE8
- Remove `clonesWork` function

Fixes #65

### CODE REVIEW

File: can-view-target.js [hunk 1]
```diff
@@ -83,56 +44,9 @@ var processNodes = function(nodes, paths, location, document){
...
-
-			while(child) {
-				copy.appendChild( cloneNode(child) );
-				child = child.nextSibling;
-			}
-		}
-
-		return copy;
-	};
+var cloneNode = function(el){
```
Comment (matthewp) [inline can-view-target.js:47]: Can we remove this function and just call `el.cloneNode(true)` wherever this function is currently used?

Comment (cherifGsoul) [reply can-view-target.js:47]: Yeah, but what if this https://github.com/canjs/can-view-target/blob/d20dab3619180104486a24793d47738b2d66952d/can-view-target.js#L289 is called elsewhere in another package?

Comment (cherifGsoul) [reply can-view-target.js:47]: Done!

Comment (matthewp) [reply can-view-target.js:47]: Sorry, I didn't see that it was being exported. For that reason I think we should keep it in for backwards compatibility. Sorry for overlooking that.

Comment (cherifGsoul) [reply can-view-target.js:47]: @matthewp it is fixed!

Comment (matthewp) [reply can-view-target.js:47]: Thanks, looks good to me then.

**Pull request merged**